features/weed_reported_location.feature

```diff
@@ -0,0 +1,68 @@
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#  ====================================================================
+
+Feature: Processing iNaturalist observations populates the CAMS iNaturalist weed location correctly
+
+On an initial synchronisation of an iNaturalist observation to CAMS, the location of the iNaturalist observation is saved to 
+the CAMS feature geolocation and to the "iNaturalist Latitude" and "iNaturalist Longitude" fields of the feature. 
+These latter two fields are referred to as the "iNaturalist location" below.
+
+After synchronising the observation to CAMS, a user may then update the geolocation of the weed feature on the CAMS map. 
+If this occurs, we don't want to overwrite the location updated by the CAMS user unless the iNaturalist user subsequently updates the location.
+The synchroniser reads the CAMS feature geolocation and compares it to the CAMS iNaturalist location fields to determine 
+whether the location has been updated by the user. If it has, the iNaturalist location will not override the CAMS geolocation.
+
+Rule: Whenever the iNaturalist location is updated, both the CAMS feature geolocation and the iNaturalist location fields are updated
+    Example: The CAMS iNaturalist location is set to the iNaturalist location for new observations
+        Given iNaturalist has a new observation at 'latitude': -41.2920665997, 'longitude': 174.7628175
+        When we process the observation
+        Then the iNaturalist location is recorded in CAMS as 'latitude': -41.2920665997, 'longitude': 174.7628175
+
+    Example: The CAMS iNaturalist location is updated when the iNaturalist location is changed
+        Given iNaturalist has an existing observation at 'latitude': -41.2920665997, 'longitude': 174.7628175
+        And that observation has been synced
+        And the geolocation is updated to 'latitude': -41.1234567, 'longitude': 174.7654321
+        When we process the observation
+        Then the iNaturalist location is recorded in CAMS as 'latitude': -41.1234567, 'longitude': 174.7654321
+    
+    Example: For an unchanged, existing CAMS feature, the CAMS geolocation and iNaturalist location are updated when the original iNaturalist location is changed
+        Given iNaturalist has an existing observation at 'latitude': -41.291111, 'longitude': 174.761111
+        And that observation has been synced
+        And the geolocation is updated to 'latitude': -41.292222, 'longitude': 174.762222
+        When we process the observation
+        Then the iNaturalist location is recorded in CAMS as 'latitude': -41.292222, 'longitude': 174.762222
+        And the CAMS geolocation is set to 'latitude': -41.292222, 'longitude': 174.762222
+
+    Example: For a changed CAMS feature, the CAMS geolocation and iNaturalist location are updated when the iNaturalist location is changed
```

Comment:
...and **CAMS** iNaturalist location...

Reply:
Please change the first `iNaturalist location` to `CAMS iNaturalist location` for clarity